crates/compiler/mono/src/inc_dec.rs

```diff
@@ -1288,6 +1288,7 @@ pub(crate) fn lowlevel_borrow_signature(op: LowLevel) -> &'static [Ownership] {
         StrGetUnsafe | ListGetUnsafe => &[BORROWED, IRRELEVANT],
         ListConcat => &[OWNED, OWNED],
         StrConcat => &[OWNED, BORROWED],
+        StrConcatUtf8 => &[OWNED, BORROWED],
```

Comment:
I think this is correct, as we are mutating the `List U8` but just copying the `Str`. But with that reasoning, I cannot explain why `ListConcat` is `OWNED, OWNED`.